src/core/runtime.d

```diff
@@ -313,13 +313,54 @@ struct Runtime
     }
 
 
+    /**
+     * Set source file path for coverage reports.
+     *
+     * Params:
+     *  path = The new path name.
+     */
+    static void coverSourcePath(string path)
```

Comment:
Are you're wondering why I didn't just document the declarations for dmd_coverSourcePath?
Well b/c they have arcane names that don't fit into the rest of core.runtime's API.
I don't really like the namespacing `Runtime` struct either, but mixing it worse.
http://dlang.org/phobos/core_runtime.html#.Runtime